.github/workflows/weekly.yml

```diff
@@ -1,8 +1,8 @@
-name: fortnightly tests
+name: weekly tests
 
 on:
   schedule:
-  - cron: 17 7 1,15 * *
+  - cron: 37 7 * * 1
```

Comment:
I just changed the `17` → `37` because I often forget which of the first two slots in a cron schedule correspond to hours and which to minutes.  This will remove the hours/minutes ambiguity when running cron jobs on all planets in the solar system, except for Mercury and Venus.